packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -459,7 +475,6 @@ export class Tooltip extends FoundationElement {
         if (!this.tooltipVisible) {
             return;
         }
-        this.viewportElement = document.body;
```

Comment:
What sets viewportElement now? I don't see it being set explicitly anywhere. Is it deferred to the anchored region and is it needed anymore?

Reply:
Falling back to the default set by anchored region.